src/geometry/geom2/clone.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Performs a shallow copy of the given geometry.
+ * @params {geom2} geometry - the geometry to make a shallow copy of
+ * @returns {geom2} new geometry
+ */
+const clone = (geometry) => Object.assign({}, geometry)
```

Comment:
can you explain why clone is not explicit anymore ? Object.assign does not handle all copying as one would expect, so it needs to be used very carefully!

Reply:
I depends on what we expect. Do we expect 'shallow' or 'deep' copies of the object?

math objects perform 'deep' copies.

Reply:
but should a clone not be deep by definition ? 
we never mutate things so I am not sure it matters in the end, but my point about the issues with Object.assign still stand sadly

Reply:
Good. We just need to come to the same understanding.

I will change to be ‘deep’ clones.

Reply:
Done. Clone performs DEEP copies of the contents. Please review.

Reply:
awesome ! I'll check the updated code

Reply:
ok seems good !